landing-zone-submodule/management-vpc/variables.tf

```diff
@@ -0,0 +1,304 @@
+  type        = string
+  sensitive   = true
+}
+
+variable "region" {
+  description = "The region to which to deploy the VPC"
+  type        = string
+  default     = "au-syd"
+}
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources"
+  type        = string
+  default     = "management"
+}
+
+variable "resource_group" {
+  type        = string
+  description = "An existing resource group name to use for this example, if unset a new resource group will be created"
+  default     = null
+}
+
+variable "tags" {
+  description = "List of tags to apply to resources created by this module."
+  type        = list(string)
+  default     = []
+}
+
+
+#############################################################################
+# VPC variables
+#############################################################################
+
+variable "network_cidr" {
+  description = "Network CIDR for the VPC. This is used to manage network ACL rules for cluster provisioning."
+  type        = string
+  default     = "10.0.0.0/8"
+}
+
+variable "classic_access" {
+  description = "Optionally allow VPC to access classic infrastructure network"
+  type        = bool
+  default     = null
+}
+
+variable "use_manual_address_prefixes" {
+  description = "Optionally assign prefixes to VPC manually. By default this is false, and prefixes will be created along with subnets"
+  type        = bool
+  default     = true
+}
+
+variable "default_network_acl_name" {
+  description = "Override default ACL name"
+  type        = string
+  default     = null
+}
+
+variable "default_security_group_name" {
+  description = "Override default VPC security group name"
+  type        = string
+  default     = null
+}
+
+variable "default_routing_table_name" {
+  description = "Override default VPC routing table name"
+  type        = string
+  default     = null
+}
+
+variable "default_security_group_rules" {
+  description = "Override default security group rules"
+  type = list(
+    object({
+      name      = string
+      direction = string
+      remote    = string
+      tcp = optional(
+        object({
+          port_max = optional(number)
+          port_min = optional(number)
+        })
+      )
+      udp = optional(
+        object({
+          port_max = optional(number)
+          port_min = optional(number)
+        })
+      )
+      icmp = optional(
+        object({
+          type = optional(number)
+          code = optional(number)
+        })
+      )
+    })
+  )
+
+  default = []
+}
+
+variable "address_prefixes" {
+  description = "Use `address_prefixes` only if `use_manual_address_prefixes` is true otherwise prefixes will not be created. Use only if you need to manage prefixes manually."
+  type = object({
+    zone-1 = optional(list(string))
+    zone-2 = optional(list(string))
+    zone-3 = optional(list(string))
+  })
+
+  default = null
+}
+
+variable "network_acls" {
```

Comment:
Just align the def with the parent one https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/blob/main/variables.tf#L98 which has been updated recently

Reply:
Using the same network acl from the parent module.